ChangeLog.md

```diff
@@ -1,3 +1,26 @@
+## [1.0.1] - 2020-05-13
+
+### Improvements
+
+  * Adjusted the Largest-First algorithm to pay for outputs collectively
+    instead of individually.
+
+    The updated algorithm should now be successful at paying for any set of
+    outputs of total value v provided that the total value u of available
+    inputs satisfies u ≥ v. The cardinality restriction requiring the number of
+    inputs to be greater than the number of outputs has been removed.
+
+    See PRs #73 and #76.
```

Comment:
Perhaps it would be better to refer to git change identifiers instead of PR numbers. (Just in case we move the repository somewhere else in future.)